src/lib/builtins/array.rs

```diff
@@ -698,25 +700,25 @@ mod tests {
     #[test]
     fn concat() {
         //TODO: array display formatter
-        let realm = Realm::create();
-        let mut engine = Executor::new(realm);
-        let init = r#"
-        var empty = new Array();
-        var one = new Array(1);
-        "#;
-        forward(&mut engine, init);
-        // Empty ++ Empty
-        let _ee = forward(&mut engine, "empty.concat(empty)");
-        //assert_eq!(ee, String::from(""));
-        // Empty ++ NonEmpty
-        let _en = forward(&mut engine, "empty.concat(one)");
-        //assert_eq!(en, String::from("a"));
-        // NonEmpty ++ Empty
-        let _ne = forward(&mut engine, "one.concat(empty)");
-        //assert_eq!(ne, String::from("a.b.c"));
-        // NonEmpty ++ NonEmpty
-        let _nn = forward(&mut engine, "one.concat(one)");
-        //assert_eq!(nn, String::from("a.b.c"));
+        // let realm = Realm::create();
+        // let mut engine = Executor::new(realm);
+        // let init = r#"
+        // var empty = new Array();
+        // var one = new Array(1);
```

Comment:
Currently, `new Array(1)` produces `[1]` (an array with one element, which is `1`), when, in fact, it should produce an array with `length` 1 and its first element should be `undefined`.

Reply:
We will pick this up in https://github.com/jasonwilliams/boa/issues/209